airflow/ui/src/router.tsx

```diff
@@ -168,4 +168,8 @@ export const routerConfig = [
   },
 ];
 
-export const router = createBrowserRouter(routerConfig, { basename: "/webapp" });
+const locationPath = globalThis.window.location.pathname;
+const indexOf = locationPath.indexOf("webapp/");
```

Comment:
Now that we are adding a base tag, is it worth using that instead of the path based guessing approach? (What we have is probably fine, but we could not have to guess at all)

Reply:
Oh, it want's the path, nm, probably not worth it